Header header
uint16 STRAIN1
float32 STRAIN1_AVG
float32 STRAIN1_VAR